backend/src/main/java/eu/bbmri_eric/negotiator/governance/resource/ResourceModelMapper.java

```diff
@@ -30,5 +38,13 @@ public void addMappings() {
         mapper -> mapper.map(Resource::getSourceId, ResourceDTO::setId));
 
     resourceToDTOTypeMap.addMappings(mapper -> mapper.map(Resource::getName, ResourceDTO::setName));
+
+    resourceToDTOTypeMap.addMappings(
+        mapper -> mapper.map(Resource::getDescription, ResourceDTO::setDescription));
+
+    resourceToDTOTypeMap.addMappings(
+        mapper -> mapper.map(Resource::getContactEmail, ResourceDTO::setContactEmail));
+
+    resourceToDTOTypeMap.addMappings(mapper -> mapper.map(Resource::getUri, ResourceDTO::setUri));
```

Comment:
are you sure these mappings are needed? Arent they handled automatically? Plus I can see you are assigning contact email to description

Reply:
Actually they aren't. Removed in 4ec4c46e1d75181b6470a308bdc7df65b774162b